node/rpc/src/lib.rs

```diff
@@ -13,7 +13,7 @@ mod rpc_util;
 mod rpc_ws_handler;
 mod state_api;
 mod sync_api;
-mod wallet_api;
+pub mod wallet_api;
```

Comment:
Is this ever used anywhere else? Does it need to be public?

Reply:
The other command modules used the raw function name and imported directly from `rpc_client`. I wanted to explicitly have `wallet_ops::..` in the function call for clarity as to where the function lives. Removed to keep the pattern with other commands